open-sphere-base/mantle/src/main/java/io/opensphere/mantle/datasources/URLSource.java

```diff
@@ -0,0 +1,12 @@
+package io.opensphere.mantle.datasources;
+
+/** Interface for data sources that have a URL. */
+public interface URLSource extends IDataSource
+{
+    /**
+     * Get the URL string.
+     *
+     * @return the URL string
+     */
+    String getURLString();

```

Comment:
I would rather it just be called getURL()

Reply:
Fixed.